pkg/jobrunaggregator/jobrunaggregatorlib/big_query.go

```diff
@@ -92,8 +94,34 @@ func (d dryRunInserter) Put(ctx context.Context, src interface{}) (err error) {
 	buf := &bytes.Buffer{}
 	fmt.Fprintf(buf, "BULK INSERT into %v\n", d.table)
 	for i := 0; i < srcVal.Len(); i++ {
-		s := srcVal.Index(i).Interface()
-		fmt.Fprintf(buf, "\tINSERT into %v: %v\n", d.table, s)
+
+		switch srcVal.Index(i).Interface().(type) {
+		case *jobrunaggregatorapi.TestRunRow:
+			s := srcVal.Index(i).Interface().(*jobrunaggregatorapi.TestRunRow)
+			fmt.Fprintf(buf, "\tINSERT into %v: prowJob=%v, ts=%v, lts=%v, tcn=%v\n", d.table, s.ProwJob.ObjectMeta.Name,
+				s.TestSuites[0], len(s.TestSuites), s.TestCase.Name)
+			if len(s.TestSuites) > 1 {
+				fmt.Println("Found a testSuite list greater than one")
+			}
+
+		case *jobrunaggregatorapi.JobRunRow:
+			s := srcVal.Index(i).Interface().(*jobrunaggregatorapi.JobRunRow)
+			fmt.Fprintf(buf, "\tINSERT into %v: name=%v, jobname=%v, status=%v\n", d.table, s.Name, s.JobName, s.Status)
+
+		case *jobrunaggregatorapi.BackendDisruptionRow:
+			s := srcVal.Index(i).Interface().(*jobrunaggregatorapi.BackendDisruptionRow)
+			fmt.Fprintf(buf, "\tINSERT into %v: BEName=%v, JobRunName=%v, DisSecs=%v\n", d.table, s.BackendName, s.JobRunName, s.DisruptionSeconds)
+
+		case jobrunaggregatorapi.JobRow:
```

Comment:
Hmm, should this be a pointer like the others?

Reply:
Good question. I found that one is actually not a pointer.  [This](https://github.com/openshift/ci-tools/blob/541d29e664365b43a93aa46f8c0c916cb460f5f4/pkg/jobrunaggregator/jobtableprimer/create_jobs.go#L44) shows that for Jobs, it's called with a slice of JobRows and not slice of pointers like the others.

Reply:
I think that's an indication that you want to handle both the pointer and non-pointer cases.